public/js/app/components/RoundInformation/RoundInformationInfoGrid.jsx

```diff
@@ -118,7 +118,7 @@ function RoundInformationInfoGrid({ courseCode, courseRound, selectedSemester })
       </Item>
       <Item title={translation.courseRoundInformation.round_target_group} html={courseRound.round_target_group} />
       <Item title={translation.courseRoundInformation.round_time_slots}>
-        <PlannedModules courseCode={courseCode} courseRound={courseRound} selectedSemester={selectedSemester} />
+        <PlannedModules plannedModules={plannedModules} />
       </Item>
```

Comment:
Now when call to `usePlannedModules` is moved up in the tree we can remove PlannedModules.jsx completly and just use the generic html-prop to <Item> here instead, like this:

`<Item title={translation.courseRoundInformation.round_target_group} html={plannedModules} />`